.circleci/config.yml

```diff
@@ -1,33 +1,39 @@
 version: 2.1
 
-jobs:
-  package-macos:
-    parameters:
-      xcode-version:
-        type: "string"
+# Define executors that various packaging jobs need.  This lets us make one
+# packaging job that accepts an executor parameter and avoids duplicating the
+# packaging steps.
+executors:
+  manylinux-2014-x86_64:
+    docker:
+      - image: "quay.io/pypa/manylinux2014_x86_64"
+  manylinux_2_24-x86_64:
+    docker:
+      - image: "quay.io/pypa/manylinux_2_24_x86_64"
 
-    macos:
-      xcode: "<< parameters.xcode-version >>"
+  # Since this executor doesn't use a pypa-defined image created specifically
+  # for building wheels, we're a little bit less confident of exactly what
+  # kind of manylinux wheel will come out of it.  x_y here is a reference to
+  # the "future-proof" (heh) PEP 600 manylinux definition.  For a given pinned
+  # image, we should probably always get the same version out (eg
+  # manylinux_2_17 as of ubuntu-2004:202101-01) but if glibc on the system is
+  # upgraded then we'll probably get higher numbers.
+  manylinux_x_y-aarch64:
+    machine:
+      image: "ubuntu-2004:202101-01"
```

Comment:
I wonder if newer versions are supported on arm (i.e. ubuntu-2004:202111-02 from [here](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images)). [This post](https://discuss.circleci.com/t/circleci-images-the-log4j-cves-december-2021/42400) suggests that version is available.

Reply:
Do we want the newest or the oldest possible version?  A manylinux_2_17 will run everywhere a manylinux_2_24 will run but the reverse is not true - right?

That aside, I think maybe I should remove the aarch64 job and try to bring it back in a follow-up.  Apart from the image choice question, there's the fact that there are no known consumers (so I'm just *assuming* this build worked) and I'm somewhat uncertain now whether we can actually trust these ubuntu images to produce correct manylinux wheels.

Reply:
Did this - see #43